src/api.ts

```diff
@@ -17,7 +17,7 @@ async function serverRequest(method: Methods, url: string, params: object, heade
 		headers,
 	}
 
-	if (json['headers']['Content-Type'] === 'multipart/form-data') {
+	if (json['headers']['Content-Type'] === 'blob') {
```

Comment:
Or you're using it just here? What's the resulting request, does it have any `Content-Type` set at all?

Reply:
yes, only to mark this request and delete it after the

Reply:
I see, but this was not supposed to work since this leaves content type detection to the server. And it will do this, but at the same time this will make our fixes from https://github.com/nspcc-dev/neofs-oauthz/pull/38/files irrelevant. Can we have a proper Content-Type set?